results/cmd/watcher/main.go

```diff
@@ -83,6 +85,7 @@ type reconciler struct {
 
 func (r *reconciler) Reconcile(ctx context.Context, key string) error {
 	r.logger.Infof("reconciling resource key: %s", key)
+	print(ctx)
```

Comment:
Avoid using the builtin print. You should be able to use r.logger instead.

Reply:
Just deleted it.